index.html

```diff
@@ -139,6 +139,11 @@
 			flex-align:center;
 			align-items:center;
 		}
+
+		.demo-frame{
```

Comment:
Should we put the background here or in the playground.html styles? Is there any reason we'd want to embed a transparent playground?